.github/workflows/python-package-conda.yml

```
@@ -13,6 +13,7 @@ jobs:
         python-version: [3.7, 3.9]
         slycot: ["", "conda"]
         pandas: [""]
+        cvxopt: ["conda"]
```

Test available and not available cvxopt

```suggestion
        cvxopt: ["", "conda"]
```